src/main/rx/rx.c

```diff
@@ -435,6 +512,50 @@ void parseRcChannels(const char *input)
     }
 }
 
+void setRssiFiltered(uint16_t newRssi, rssiSource_e source)
+{
+    if (source != rssiSource) {
+        return;
+    }
+
+    rssi = newRssi;
+}
+
+#define RSSI_SAMPLE_COUNT 16
+#define RSSI_MAX_VALUE 1023
+
+void setRssiUnfiltered(uint16_t rssiValue, rssiSource_e source)
```

Comment:
Since we are introducing this function it would be good to make other RSSI code to use it as well (channel/analog)